src/layout/admin/src/Admin.fixture.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import Admin from './';
+
+const longContent = (
```

Comment:
These are kind of annoying. Considering maybe adding `faker` for doing these things.